hw5/hw5/ViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ViewController.swift
+//  hw5
+//
+//  Created by Илья Мунгалов on 15.03.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Создать словарь, где ключ - название города/страны, значение - интересный факт и континент, где распорожен город/страна (подумайте, какой тип данных лучше использовать для значения).
+        var cityDictionary: [String: [String]]
+        cityDictionary = [
+            "Moscow": ["Сapital of Russia", "Eurasia"],
+            "Minsk": ["Сapital of Belarus", "Eurasia"],
+            "NY": ["Finance center", "North America"],
+            "Beijing": ["Сapital of China", "Eurasia"]
+        ]
+        
+        //1) Добавить новый элемент в словарь, поменять два существующих элемента в словаре, удалить один элемент из словаря.
+        cityDictionary["Capetown"] = ["Сapital of South Africa", "Africa"]
+        print(cityDictionary)
+        
+        cityDictionary["Moscow"] = ["Financial center of Russia", "Eurasia"]
+        cityDictionary["NY"] = ["The Big Apple", "North America"]
+        print(cityDictionary)
+        
+        cityDictionary["NY"] = nil
+        print(cityDictionary)
+        
+        //2) Создать отдельный словарь с названием городов/стран с длиной более 6 букв.
+        let countryDictionary = [
+            "Georgia",
+            "Azerbaijan",
+            "United Kingdom"
+        ]
+        
+        print(countryDictionary)
+        
+        //3) Создать массив интересных фактов в формате (“название города/страны: факт”), отсортировав их по длине строки по убыванию.
+        let interestingFactsArray = ["Capetown: Сapital of South Africa", "Moscow: Financial center of Russia", "Minsk: Сapital of Belarus", "Beijing: China"]
```

Comment:
The same here. We need to concat continent and fact programmatically from the initial dictionary because they should be as two separate elements and we need them as one string